### ISSUE TITLE
Scrape without logging in/without account

### ISSUE THREAD

**Issue by yehoshuadimarsky** [rarcega/instagram-scraper#222]
Hey, thanks for the great tool. I want to simply scrape one account, and don't have an Instagram account. Any way to use the tool without an account? I tried running it and omitting the `--login-user` and `--login-pass` parameters but it gives me an error that they are required. Why is that?
Thanks.

**Comment by sarsiz** [comment]
I don't think it will work. Creating an account is damn easy. You can use fake email if you are afraid of giving your primary email id.

**Comment by Emalton** [comment]
When you don't specify credentials it doesn't use them now.
